org.flatpak.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+
+  <id>org.flatpak.am-okay</id>
+  
+  <name>am-okay</name>
+  <summary>The `am-okay` program allows dynamically to copy/move files/directories (via the terminal)</summary>
+  
+  <metadata_license>MIT</metadata_license>
```

Comment:
```suggestion
  <metadata_license>CC0-1.0</metadata_license>
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#license

Reply:
Done !